help.md

```diff
@@ -0,0 +1,901 @@
+    techniques a specific adversary has been known to use. Layers can be
+    created interactively within the Navigator or generated programmatically
+    and then visualized via the Navigator.
+</p><br>
+<p>
+    Bug reports and feature requests can be submitted to our <a href="https://github.com/mitre-attack/attack-navigator/issues">GitHub Issue Tracker</a>.
+    The source code for the ATT&CK Navigator can be retrieved from our
+    <a href="https://github.com/mitre-attack/attack-navigator">GitHub repository</a>.
+</p>
+
+
+<!-- sections -->
+<span class="layers"></span>
+# Layers
+<p>
+    A layer constitutes a view of the tactics and techniques matrix for a
+    specific technology domain. Version {{nav_version}} of the Navigator can manipulate
+    either the Enterprise, Mobile, or ICS
+    ATT&CK technology domain knowledge bases. Within a technology domain, the
+    Navigator allows you to filter your view of the matrix in a variety of
+    ways, displaying the tactics and techniques that are important to you.
+</p><br>
+<p>
+    You can view the definition of any technique in the visible matrix by
+    right-clicking on the technique and selecting "view technique" in the
+    pop-up menu. A new browser tab will be opened displaying the definition of
+    the technique. In this way the Navigator allows you to explore a given
+    ATT&CK matrix and access the definitions of the techniques.
+</p><br>
+<p>
+    Beyond the filters, layers also provide a means to customize your view of
+    the matrix. To that end you can
+    <span class="section-link linkName-assigning_manual_colors">color</span>,
+    <span class="section-link linkName-hiding_disabled_techniques">hide</span>,
+    <span class="section-link linkName-adding_comments_to_techniques">comment</span>,
+    and assign <span class="section-link linkName-Scoring_Techniques">numeric scores</span>
+    to techniques to aid in analysis of threats and your defenses
+    against those threats. As stated earlier, the Navigator is designed to be
+    simple, allowing you to assign whatever meaning you want to the
+    color-codings, scores, and comments. This way the Navigator can support
+    whatever you want to do without requiring changes to the Navigator code
+    itself.
+</p><br>
+<p>
+    Each layer created is independent of other layers. However, layers can be
+    <span class="section-link linkName-Creating_Layers_from_Other_Layers">combined</span>
+    in ways to support analysis, or
+    <span class="section-link linkName-Saving_and_Loading_Layers">saved locally</span>. Layer files are
+    saved in easy to parse and easy to generate JSON file so that ATT&CK data
+    can be used in other applications, analyzed beyond the capability of the
+    ATT&CK Navigator, and generated by tools for import into the Navigator.
+    The Layer file format is described <a href="assets/NavigatorLayerFileFormatv4_1.pdf">here</a>.
```

Comment:
This hyperlink doesn't work when viewed in GitHub since the relative link to `assets` is incorrect when viewed in that context.

Reply:
Speaking of, it may be worth replacing the PDF version of the layer file format (which we always forget to update 😆 ) with a rendered copy of the actual document from the `/layers` folder.

Reply:
I think this is the only thing left in the story I haven't touched on yet. I can replace this link to the markdown version in the [/layers](https://github.com/mitre-attack/attack-navigator/blob/master/layers/LAYERFORMATv1.md) folder instead. Should I have it rendered in-app as well?

Reply:
Yes, I think it would be best as rendered in-app. Since the help page is in a dialog I'm not sure the best way to do that (close the original dialog and open the new one? Replace the dialog contents?) since spawning a dialog on top of a dialog is bad UX.

We'll still need to update the settings for which markdown file to render when we update the layer file, unless we can do something fancy with the layer version in globals.ts to infer which file to load. But at least we won't have to maintain a PDF version of the layer spec anymore.

Reply:
@isaisabel I added a new component to dynamically display the correct layer information markdown, based on the global nav ver: https://github.com/mitre-attack/attack-navigator/tree/feature/%23179-help-page/nav-app/src/app/layer-information

Let me know what you think!